### PR TITLE
developed toggle switch

### PR DESCRIPTION
**Pull request by Rehan-Khan-Engineer**

### CODE REVIEW

**Review by Abbas4tech — APPROVED** [review]

File: tasks-by-users/Rehan-Khan-Engineer/07-Toggle-switch-css/style.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/07-Toggle-switch-css/style.css:4]: Apply these properties in universal selector @Rehan-Khan-Engineer

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

**Review by shaikh-7abish — APPROVED** [review]

File: tasks-by-users/Rehan-Khan-Engineer/07-Toggle-switch-css/style.css (hunk 1 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/07-Toggle-switch-css/style.css:4]: Done @Abbas4tech

**Review by Rehmatulla-Khan — APPROVED** [review]

**Review by Rehmatulla-Khan — APPROVED** [review]

**Review by mmusk — APPROVED** [review]
LGTM

**Comment by Rehan-Khan-Engineer** [comment]
Hi @Abbas4tech , could you please look again and approve now! I have done the changes

**Comment by Rehan-Khan-Engineer** [comment]
> _No description provided._

Hi @Abbas4tech , Could you please approve this. I have done the changes.

**Comment by Rehmatulla-Khan** [comment]
close this PR @Rehan-Khan-Engineer

**Review by Naufil-Shemle — APPROVED** [review]
Say Abbas to review the code again so that you can merge

**Review by Abbas4tech — APPROVED** [review]

**Pull request merged**